### DISCUSSION TITLE
调整目录中的序号宽度

### DISCUSSION THREAD

**Discussion by RoserX** [nju-lug/NJUThesis#258] [Q&A]
请问我使用
\ctexset{
  subsection/name = {,.},
  subsection/number = \arabic{subsection}
}
修改目录后，三级标题在目录里后面会多出空格怎么办
![%7O$M}O)$TU2H`@}UY0(%0I](https://github.com/nju-lug/NJUThesis/assets/172917978/b71b25f2-9d01-44c9-8026-b0824750db93)

**Comment by atxy-blip** [comment]
可以在导言区打个补丁。标准文档类给这个序号盒子的默认宽度是 `3.2em`（第三个参数），可以改成一个更小的值，比如 `2em`。

```tex
\makeatletter
\renewcommand*\l@subsection{\@dottedtocline{2}{3.8em}{3.2em}}
\makeatother
```
